hack/test-end-to-end-docker.sh

```diff
@@ -15,9 +15,8 @@ echo "[INFO] Starting containerized end-to-end test"
 
 unset KUBECONFIG
 
-TMPDIR="${TMPDIR:-"/tmp"}"
-BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-e2e-containerized}"
 setup_env_vars
+setup_temp_dir_vars "test/end-to-end/containerized/"
```

Comment:
Well if they never get to a customer I don't care.  Any time you have a privileged script writing to /tmp, there is a risk an evil user could do something to get priv escalation.  So if you are privileged use /run and protect yourself.

Wrote this back in 2007.

http://danwalsh.livejournal.com/11467.html?thread=80075

Most likely does not apply to you guys, but if I see an email talking about /tmp, I usually chime in.